apps/sms/desktop-mock/navigator_moz_mobile_connections_shim.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
We didn't have the navigator_moz_mobile_connections for the desktop mock originally because it's not necessary in desktop mode. So there's no corresponding navigator_moz_mobile_connections.js actually. Not sure if you would prefer to have a navigator_moz_mobile_connections.js even it's not in used now.